visidata/vdobj.py

```diff
@@ -166,3 +168,24 @@ def screenHeight(self):
     @property
     def screenWidth(self):
         return self.scrFull.getmaxyx()[1] if self.scrFull else 80
+
+    @property
+    def activeCommand(self):
+        '''Returns the active command for the current thread.
+            Returns a CommandLogRow, or None.
+            Returns None when no command has ever been stored for the current thread.'''
+        tid = threading.get_ident()
+        if tid not in self.activeCommands:
+            return None
+        return self.activeCommands[tid]
+
+    @activeCommand.setter
+    def activeCommand(self, value):
+        self.setThreadActiveCommand(threading.current_thread(), value)
+
+    def setThreadActiveCommand(self, thread, value):
```

Comment:
Yes, that's the only place it was used. I left the thread-specifying function there in case we wanted to call it explicitly in `_with_active_cmd` for clarity. Because as I have it now, it is a bit strange to see `cmd = vd.activeCommand` and then, two lines later `if cmd: vd.activeCommand = cmd`.

I've merged it.

Reply:
I see what you mean.  We'll add a comment there referring to this issue/PR.